packages/ui/src/content/Icon.tsx

```diff
@@ -85,4 +90,10 @@ export const HeroIcons = {
   User: wrapHeroIcon(UserIcon),
   Bars3: wrapHeroIcon(Bars3Icon),
   EllipsisVertical: wrapHeroIcon(EllipsisVerticalIcon),
+  Activity: wrapHeroIcon(QueueListIcon),
+  Settings: wrapHeroIcon(Cog8ToothIcon),
+  Feedback: wrapHeroIcon(ChatBubbleBottomCenterTextIcon),
+  ChevronRight: wrapHeroIcon(ChevronRightIcon),
+  Menu: wrapHeroIcon(Bars3Icon),
+  Interaction: wrapHeroIcon(ArrowsRightLeftIcon),
```

Comment:
I kept all the icon names the same as hero icons (so it's easy to find). But not sure if we want that. Just FYI that that pattern is now broken